api/logics.py

```diff
@@ -882,7 +880,6 @@ def cancel_order(cls, order, user, state=None):
             # adds a timeout penalty
             cls.cancel_bond(order.taker_bond)
             cls.kick_taker(order)
-            # send_message.delay(order.id,'taker_canceled_b4bond') # too spammy
```

Comment:
I'm not a fan of leaving commented code, do you want to just check how it goes?

Reply:
Agree,

I had commented out this line long time ago. I was not sure the solution we found was good enough. But now that I revisited this file, I think it's clear the solution worked out and it is time to delete this commented code, forever (actually, this PR also deletes the function this line is calling to) :D